src/org/apache/xerces/stax/events/EntityDeclarationImpl.java

```diff
@@ -40,9 +40,13 @@ public final class EntityDeclarationImpl extends XMLEventImpl implements
     private final String fNotationName;
 
     /**
-     * @param eventType
-     * @param location
-     * @param schemaType
+     * Constructor for an implementation of {@link EntityDeclaration}.
+     *
+     * @param publicId the publicId
```

Comment:
don't just repeat the argument name; might as well leave it out in that case

Reply:
fixing now to align with Javadoc in the interface